apps/greenhouse/package.json

```diff
@@ -45,12 +51,6 @@
     "clean:cache": "rm -rf .turbo"
   },
   "dependencies": {
-    "@cloudoperators/juno-app-doop": "*",
-    "@cloudoperators/juno-app-heureka": "*",
-    "@cloudoperators/juno-app-supernova": "*",
-    "@cloudoperators/juno-messages-provider": "*",
-    "@cloudoperators/juno-oauth": "*",
```

Comment:
I think we still need 
- @cloudoperators/juno-ui-components
- @cloudoperators/juno-oauth
- @cloudoperators/juno-messages-provider
or?

Reply:
and why not in dependencies???